clang/lib/CodeGen/TargetInfo.cpp

```diff
@@ -8496,16 +8496,16 @@ void TCETargetCodeGenInfo::setTargetAttributes(
 
         SmallVector<llvm::Metadata *, 5> Operands;
         Operands.push_back(llvm::ConstantAsMetadata::get(F));
-        unsigned XDim = Attr->getXDimVal()->getZExtValue();
-        unsigned YDim = Attr->getYDimVal()->getZExtValue();
-        unsigned ZDim = Attr->getZDimVal()->getZExtValue();
 
-        Operands.push_back(llvm::ConstantAsMetadata::get(
-            llvm::Constant::getIntegerValue(M.Int32Ty, llvm::APInt(32, XDim))));
-        Operands.push_back(llvm::ConstantAsMetadata::get(
-            llvm::Constant::getIntegerValue(M.Int32Ty, llvm::APInt(32, YDim))));
-        Operands.push_back(llvm::ConstantAsMetadata::get(
-            llvm::Constant::getIntegerValue(M.Int32Ty, llvm::APInt(32, ZDim))));
+        Operands.push_back(
```

Comment:
Are you reverting to community code here?

Reply:
Yes! See https://github.com/llvm/llvm-project/blob/main/clang/lib/CodeGen/TargetInfo.cpp#L8575.